automated-comments/ruby/general/incorrect_indentation.md

```diff
@@ -0,0 +1 @@
+Use two spaces to indent this code. In Ruby, using two spaces is a strong community standard - we never use tabs, and always only use two spaces.
```

Comment:
```suggestion
Use two spaces to indent this code. In Ruby, using two spaces is a strong community standard - never use tabs, always use two spaces.
```

Without the persona this is more of a  "never, always" rule instead of "we, the community". Idk. ignore if you hate 🗡